Fix #43 - Design Login Screen UI

nishank95 opened the pull request:
### Description
Added Login Screen UI as per the designed Mockup for Login Screen

Fixes #43 

### Type of Change:
**Delete irrelevant options.**

- User Interface
Changed the UI for Login Screen Activity.
Added the following drawable files:
  1. Background for Login Screen
  2. ic_phone image for edit text drawable
  3. Background for proceed button


### Screenshot

BEFORE:
N/A

AFTER:
<img src="https://user-images.githubusercontent.com/25129671/39387469-610b4f28-4a97-11e8-85d8-3c60b057f5af.jpeg" height="400" alt="login_screen">


**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)



### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.
N/A

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

nishank95 commented:
1. I will make the changes in style of action bar
2. Because when the device rotates the form elements are not accessible and hence used scroll view.

Rondaul commented:
@nishank95  Use the scrollview in lanscape layout of activity_login. You can create lanscape by clicking on the orientation in the editior. Keep the scroll view there

harikautilya commented on app/src/main/res/layout/activity_login.xml:
```diff
@@ -1,16 +1,151 @@
-        tools:context=".ui.login.LoginActivity">
 
-    </android.support.constraint.ConstraintLayout>
+        <ScrollView
```
Why do we need scroll view ? it's a single page view.If u wanna push the screen up on keyboard opening use adjust pan in manifest file. 
https://stackoverflow.com/questions/17410499/difference-between-adjustresize-and-adjustpan-in-android

nishank95 replied:
Scroll view is used if phone is rotated

harikautilya replied:
Develop the application only in portrait mode

nishank95 replied:
Changed the LoginActivity to portrait mode

harikautilya commented on app/src/main/res/layout/activity_login.xml:
```diff
@@ -1,16 +1,151 @@
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+           >
+
```
also pull from master and use the ids supplied

nishank95 replied:
can you specifiy which id because in master branch>activity_login.xml file i cant find any id's

harikautilya replied:
Please pull from master for the ids

harikautilya commented on app/src/main/res/layout/activity_login.xml:
```diff
@@ -1,16 +1,151 @@
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent" />
+
```
Use defined Id here as well

harikautilya commented on app/src/main/res/layout/activity_login.xml:
```diff
@@ -1,16 +1,151 @@
+                    app:layout_constraintHorizontal_bias="0.503"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/tv_app_title" />
+
```
Use defined Id here as well

harikautilya commented on app/src/main/res/layout/activity_login.xml:
```diff
@@ -1,16 +1,151 @@
+                        app:layout_constraintEnd_toStartOf="@+id/tv_label_or"
+                        app:layout_constraintStart_toStartOf="parent"
+                        tools:layout_editor_absoluteY="354dp" />
+
```
Use defined Id here as well

Rondaul commented:
@nishank95 please put the scrollview only in landscape mode. no need in the portrait mode

nishank95 commented:
@Rondaul I have added a land/activity_login.xml for landscape mode which contains the scroll view and removed scroll view from portrait mode.If it's alright please let me know then i will commit the changes

Aaklii commented:
Let me know when it needs to be merged

Aaklii commented:
@Kautilyahari @geekanamika @Rondaul @vvkvivekl 
Review the file before  I merge them/

Rondaul commented:
@Aaklii tell him to use the ids provided in `ids.xml`

nishank95 commented:
Hey @Rondaul I have used the id's from ids.xml file you can check activity_login.xml file.
